ARCHITECTURE.md

```diff
@@ -120,14 +120,15 @@ Update instructions:
 
 ### SDK/CLI/Wasm top-level crates
 
-| Crate     | Description                          |
-| --------- | ------------------------------------ | ------------------------ | --- |
-| rerun-cli | Rerun native CLI binary crate        |
-| Rerun     | Rerun Rust SDK and Viewer shim crate |
-| rerun_c   | Rerun C SDK                          |
-| rerun_py  | Rerun Python SDK                     |
-| re_sdk    | Rerun logging SDK                    |
-| <!--      | re_renderer_examples                 | Examples for re_renderer | --> |
```

Comment:
It appears it predates this PR, but the markdown rendering for this table is broken. It would be great to fix it.

<img width="929" height="126" alt="Image" src="https://github.com/user-attachments/assets/edec2e8a-e1a9-499a-8fbf-167b5c2aab7d" />